catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -108,35 +104,36 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
     try (Connection connection = getConnection(databaseName)) {
-      String comment;
-      Map<String, String> properties;
-      try (ResultSet table = getTable(connection, tableName)) {
-        if (!table.next()) {
-          throw new NoSuchTableException("Table " + tableName + " does not exist.");
-        }
-        comment = table.getString("REMARKS");
-        properties = extractPropertiesFromResultSet(table);
+      // 1.Get table information
+      ResultSet table = getTable(connection, databaseName, tableName);
+      if (!table.next() || !tableName.equals(table.getString("TABLE_NAME"))) {
+        throw new NoSuchTableException(
+            String.format("Table %s does not exist in %s.", tableName, databaseName));
       }
-      JdbcColumn[] jdbcColumns;
-      try (ResultSet column = getColumns(connection, tableName)) {
-        List<JdbcColumn> result = new ArrayList<>();
-        while (column.next()) {
-          result.add(extractJdbcColumnFromResultSet(column));
-        }
-        jdbcColumns = result.toArray(new JdbcColumn[0]);
+
+      // 2.Get column information
+      List<JdbcColumn> jdbcColumns = new ArrayList<>();
+      ResultSet columns = getColumns(connection, databaseName, tableName);
+      while (columns.next()) {
+        JdbcColumn.Builder columnBuilder = getBasicJdbcColumnInfo(columns);
+        boolean autoIncrement = getAutoIncrementInfo(columns);
+        columnBuilder.withAutoIncrement(autoIncrement);
+        jdbcColumns.add(columnBuilder.build());
       }
-      return new JdbcTable.Builder()
-          .withName(tableName)
-          .withColumns(jdbcColumns)
-          .withComment(comment)
-          .withProperties(properties)
-          .withAuditInfo(AuditInfo.EMPTY)
-          .build();
-    } catch (final SQLException se) {
-      throw this.exceptionMapper.toGravitinoException(se);
+
+      JdbcTable.Builder jdbcTableBuilder = createJdbcTableBuilder(table);
+
+      // 3.The underlying implementation class customizes the extension of table information.
+      extractJdbcTable(connection, tableName, jdbcTableBuilder);
```

Comment:
It's more clear to split `extractJdbcTable` into 3 functinons:
1. getTableProperties()
2. getIndexs()
3. fill up the meta couldn't find from driver, I couldn't figure out a property name.